srcCxx/ShapeType_main.cxx

```diff
@@ -219,7 +221,7 @@ int run(DomainId_t domain_id, bool use_security,
     Topic *sub_topic = NULL;
     ReturnCode_t retcode;
 
-    DomainParticipant *participant = ShapeTypeConfigurator::create_participant(domain_id, use_security, governance_file, permissions_file, enable_logging);
+    DomainParticipant *participant = ShapeTypeConfigurator::create_participant(domain_id, use_security, governance_file, permissions_file, key_establishment_algorithm, enable_logging);
```

Comment:
Since this changes the API between the common code in ShapeType_main.cxx and the implementation-specific code in the subdirectories, I suggest having this PR update the code for other (non-RTI) implementations so that their `create_participant` functions take the additional `const char*` parameter and ignore it.  Otherwise this is going to be a breaking change.  I realize you don't have development environments set up for all other implementations, but this should be a trivial fix.